corehq/apps/hqcase/management/commands/ptop_fast_reindexer.py

```diff
@@ -172,14 +177,10 @@ def load_from_view(self):
                 fout.write('{}\n'.format(simplejson.dumps(row)))
         self.log("View and sequence written to disk: %s" % datetime.utcnow().isoformat())
 
-    def load_seq_from_disk(self):
-        """
-        Main load of view data from disk.
-        """
+    def _load_seq_from_disk(self):
         self.log("Loading from disk: %s" % datetime.utcnow().isoformat())
         with open(self.get_seq_filename(), 'r') as fin:
-            current_db_seq = fin.read()
-            self.pillow.set_checkpoint({'seq': current_db_seq})
```

Comment:
pulled this out of the function after it was called because it seemed confusing that a function with this name would set the checkpoint